README.md

```diff
@@ -261,6 +261,24 @@ If you want to build a specific module later on, you can do the same but with ju
 module you wanna build. It'll get added to the same directory where the others are.
 Or, if you are particularly lazy, just add the module name to the make invocation.
 
+## FISh
+The `sled.fish` FiFo supports the following commands:
+
+| Command | Action |
+| --- | --- |
+| `/blank` | Blank the output |
+| `/error42` | Exit the program |
+| `/next` | Jump to the next effect |
```

Comment:
*Technically*, there is no `/next`. It's treated the same as any other unknown command.

Reply:
I know, but thats imho not important here (and a practical example)?